server/utils/createObjectFromQuery.js

```diff
@@ -7,7 +7,7 @@ const createObjectFromQuery = (data) => {
         filteredValues['name.hebrew'] = name.hebrew
       }
       if (name.english) {
-        filteredValues['name.english'] = name.english
+        filteredValues['name.english'] = new RegExp(`^${name.english}`, 'i');
```

Comment:
why not use includes?

Reply:
Here I am creating on object, that I then can pass as a filter in any find() or findOne() or findOneAndUpdate() of Mongoose, to query by this object.
I define here the fields of the object 'filteredValues' and not making any comparison.